source/MaterialXRuntime/RtNodeGraph.cpp

```diff
@@ -11,12 +11,14 @@
 
 #include <MaterialXRuntime/Private/PvtPrim.h>
 
+#include <MaterialXCore/Element.h>
+
 namespace MaterialX
 {
 
 namespace
 {
-    const RtToken SOCKETS("_nodegraph_internal_sockets");
+    static const RtToken SOCKETS("_nodegraph_internal_sockets");
```

Comment:
Nitpick: the static keyword is not really needed in this case since in an anonymous namespace the object is unreachable from other translation units, so effectively has internal linkage already.